jekyll/_cci2/status.md

```diff
@@ -30,3 +30,6 @@ CircleCI provides an integrated dashboard with the status of your build jobs:
 - RUNNING: Job run is in progress
 - ON HOLD: Job is waiting for approval
 - QUEUED: Job is waiting for available container
+
+If you are using [workflows]({{ site.baseurl}}/2.0/workflows-overview) you may
```

Comment:
ah sorry, actually this should link to /2.0/workflows/#overview that is the best doc to point to